FORMULA

```diff
@@ -1,9 +1,9 @@
 name: prometheus
-os: FreeBSD
-os_family: FreeBSD
-version: 3.3.0
+os: Debian, Ubuntu, Raspbian, RedHat, Fedora, CentOS, Amazon, Suse, openSUSE, Gentoo, Funtoo, Arch, Manjaro, Alpine, FreeBSD, OpenBSD, Solaris, SmartOS, Windows, MacOS
+os_family: Debian, RedHat, Suse, Gentoo, Arch, Alpine, FreeBSD, OpenBSD, Solaris, Windows, MacOS
+version: 0.2.2
```

Comment:
Shouldn't touch the `version` once `semantic-release` is already running for a formula.

Reply:
okay, I'll fix this